lerobot/scripts/push_dataset_to_hub.py

```diff
@@ -236,19 +236,25 @@ def push_dataset_to_hub(
             fname = f"{key}_episode_{episode_index:06d}.mp4"
             shutil.copy(videos_dir / fname, tests_videos_dir / fname)
 
-    if not save_to_disk and out_dir.exists():
+    if not save_to_disk and output_data_dir.exists():
         # remove possible temporary files remaining in the output directory
-        shutil.rmtree(out_dir)
+        shutil.rmtree(output_data_dir)
 
 
 def main():
     parser = argparse.ArgumentParser()
 
     parser.add_argument(
-        "--data-dir",
+        "--input-data-dir",
+        type=Path,
+        required=True,
+        help="Root directory containing input raw datasets (e.g. `data` or `tmp/data` or `/tmp/lerobot/data`).",
```

Comment:
What do you think of this API?
```suggestion
        help="Directory containing input raw datasets (e.g. `data/aloha_mobile_chair_raw` or `data/pusht_raw`).",
```